sharding-proxy/sharding-proxy-transport/sharding-proxy-transport-mysql/src/main/java/org/apache/shardingsphere/shardingproxy/transport/mysql/packet/command/MySQLCommandPacketType.java

```diff
@@ -253,6 +256,17 @@
      * @see <a href="https://dev.mysql.com/doc/internals/en/com-reset-connection.html">COM_RESET_CONNECTION</a>
      */
     COM_RESET_CONNECTION(0x1f);
+
+    /**
+     * The cache map for MySQLCommandPacketType.
+     */
+    private static final Map<Integer, MySQLCommandPacketType> MYSQL_COMMAND_PACKET_TYPE_CACHE = new HashMap<Integer, MySQLCommandPacketType>() {
```

Comment:
The second <Integer, MySQLCommandPacketType> can be omitted.

Reply:
done

Reply:
it's seem not friendly to IDE, like a `CASE WARNING`

Reply:
> WARNING

What does it mean? Can you explicitly point out the problem and suggestion?

Reply:
![image](https://user-images.githubusercontent.com/6848593/64942175-fb0ce400-d89a-11e9-81eb-5ba9a529e82e.png)

this is anyomous inner class, if omitted the type, a uncheck warning. i think keep it more better.

Reply:
@yanickxia
You are right. Effectively <> turns off the type checking, rather than providing type inference. An anonymous class stores the actual generic type and so you have to provide it.

@yanyzy I'm afraid you have to recover the line 263 to origin, you had already made it perfect.

Reply:
@tuohai666 
It's ok, i have changed it back.